assets/js/modules/analytics/components/setup/SetupFormGA4.js

```diff
@@ -50,6 +50,8 @@ export default function SetupFormGA4() {
 	const { setValues } = useDispatch( CORE_FORMS );
 	const { setUseSnippet } = useDispatch( MODULES_ANALYTICS_4 );
 
+	const shouldShowAssociatedPropertyNotice = useSelect( ( select ) => select( MODULES_ANALYTICS_4 ).getPropertyID() );
```

Comment:
We may want to actually assign this to a `ga4Property` variable first and then include the same conditions as below `accountID && accountID !== ACCOUNT_CREATE && ga4PropertyID`. These should be consistent between all three scenarios:

1. UA scenario: require account not empty and not create, and require UA property not empty
2. GA4 scenario: require account not empty and not create, and require GA4 property not empty
3. GA4-transitional scenario: require account not empty and not create, and require UA property _or_ GA4 property not empty

Reply:
👍